docs/tools.md

```diff
@@ -168,3 +168,54 @@ static get onPaste() {
 > Both `onPaste.handler` and `onPaste.patternHandler` can be `async` or return a `Promise`.
 
 ### Sanitize
+
+CodeX Editor provides [API](https://github.com/codex-team/codex.editor/blob/master/docs/sanitizer.md) to sanitize your fields. We recommend to clean your taint 
+string before you return your data.
+
+Call API method
+
+```javascript
+save() {
+  return {
+    text: this.api.sanitizer.clean(yourText, {}) 
+  }
+}
+
+```
+
+Editor's Core automatically checks each Block and gets sanitize rules
```

Comment:
If you pass the sanitizer config, CodeX Editor will automatically sanitize your saved data.